packages/web/src/components/data-entry/StemsAndDownloadsTriggerLegacy.tsx

```diff
@@ -0,0 +1,296 @@
+import { useCallback, useMemo } from 'react'
```

Comment:
is this adding back an old file that was deleted?

Reply:
nah. this new